### ISSUE TITLE
4.3 Tracking Update

### ISSUE THREAD

**Issue by werewolfboy13** [Squad-Wiki/squad-wiki-pipeline-map-data#50]
I will post updates here as the workflow progresses.

With the faction changes there is alot that I need to go through. Expect a few days turn around.

**Comment by werewolfboy13** [comment]
Project is not started yet, will update once it is ready.

**Comment by werewolfboy13** [comment]
![image](https://user-images.githubusercontent.com/43587009/226078054-3234349a-b6f1-4e25-b5ea-400095224bce.png)

This is a breaking change.  

@Shanomac99 I will need your assistance with this. I'll make another issue.

**Comment by werewolfboy13** [comment]
This now dependent on #51

**Comment by werewolfboy13** [comment]
Running Initial Tests Now.

**Comment by werewolfboy13** [comment]
![image](https://user-images.githubusercontent.com/43587009/226224744-1b7b5691-de6d-4c1e-8852-33a613dd64bc.png)
Initial tests prove valid. 

Will attempt to run over the next few days.

**Comment by werewolfboy13** [comment]
First pass running.
